beancount/parser/lexer.l

```diff
@@ -573,3 +577,27 @@ void build_lexer_error_from_exception()
                         "Internal error: No exception");
     }
 }
+
+int PyFile_Read(PyObject *file, char *buf, size_t max_size)
```

Comment:
Avoid naming functions that we define ourselves using the Python C API convention, it'll get confusing. Prefer to use lowercase convention used in our C code, e.g., read_from_file().